src/sass/_typography.scss

```diff
@@ -106,16 +106,24 @@
   line-height: 24px;
 }
 
-@mixin subject-areas() {
+@mixin article-review-timeline-text() {
   color: settings.$color-text;
   font-family: settings.$font-family-primary;
-  font-size: 12px;
-  line-height: 18px;
+  font-size: 14px;
+  line-height: 32px;
 }
 
-@mixin article-review-timeline-text() {
-  color: settings.$color-text;
+@mixin summary-heading() {
+  color: settings.$color-text-inactive;
   font-family: settings.$font-family-primary;
   font-size: 14px;
-  line-height: 32px;
+  line-height: 30px;
```

Comment:
@chris-huggins There are various line-heights for 14px font size. Is this correct as we 24px, 26px, 30px and 32px for 14px font size